### PR TITLE
Remove empty space below data in hexedit. Change scrolling.

### LINKED ISSUE #375: Hexview should render as many rows as possible
Currently, there's a horizontal line which could be replaced by displaying additional bytes.
Example

### PR DESCRIPTION
**Pull request by chivay**
Fixes #375

### CODE REVIEW

**Comment by mkow** [comment]
I'm also not sure whether we want that longer scroll thing. I'm used to jumping to the end of file using the scrollbar, which in this version results in seeing only the last row. Any good use-case in favor of keeping this change?

File: include/ui/hexedit.h [hunk 1]
```diff
@@ -105,6 +105,7 @@ class HexEdit : public QAbstractScrollArea {
   /** Number of rows displayed on the screen (calculated from window and font
    * height) */
   qint64 rowsOnScreen_;
+  qint64 totallyVisibleRows_;
```
Comment (mkow) [inline include/ui/hexedit.h]: Should be `totally_visible_rows_`, unfortunately the old code doesn't use Google style (I'll refactor it someday).

**Comment by mkow** [comment]
Interesting: it works well now when I press Ctrl+End, but not when I press Ctrl+Shift+End (the last line is cropped by few pixels in the latter case).

**Comment by mkow** [comment]
Same for manually moving cursor downwards.

**Comment by chivay** [comment]
Everything seems to be working right now.

**Pull request merged**